generators/app/index.js

```diff
@@ -56,8 +56,10 @@ module.exports = class extends Generator {
   }
 
   install() {
-    if (this.options['skip-install']) {
-      return;
+    if (!this.options['skip-install']) {
+      this.installDependencies({
```

Comment:
Looks like this change isn't equivalent to what the code was doing before. Is there a reason?